R/outputs.R

```diff
@@ -977,8 +980,17 @@ save_output <- function(filename, dir,
   }
 
   if (export_datapack) {
+
+    if (!is.null(vmmc_path)) {
+      assert_scalar_character(vmmc_path)
+      ## Skip the first row, the file has two rows of headers
+      vmmc_datapack <- openxlsx::read.xlsx(vmmc_path, sheet = "Datapack inputs",
+                                           startRow = 2)
+      ## TODO: Add it to relevant place in download
+    }
```

Comment:
@r-ash -- note that I added a commit to move this read file inside the `if (export_datapack)`.

Let me know if any issue with that.